### DISCUSSION TITLE
python script for sending the same event multiple times concurrently to check latency

### DISCUSSION THREAD

**Discussion by nsrawat009** [miguelgrinberg/python-socketio#1345] [Q&A]
Hi @miguelgrinberg  is there any way to  send the same event multiple times(lest say 10 requests) concurrently to calculate average latency of responses in socketio???

**Comment by miguelgrinberg** [comment]
You can, if you do it yourself. The library does not offer this as a feature, so you will need to implement the concurrent requests in your application.

**Reply by nsrawat009** [reply]
Can you provide any input for how to implement it. Iam trying to use threadpool executor but not able to send the requests, its  not working.

**Reply by miguelgrinberg** [reply]
How does it not work? There is really nothing special about this, you just need to use a solution that is compatible with your chosen async_mode.
